ui/src/components/Form/FileField/index.tsx

```diff
@@ -0,0 +1,70 @@
+import classNames from 'classnames/bind';
```

Comment:
This file has been renamed I believe. Please checkout this branch locally, and merge master back into it.